<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CountDisplay</class>
 <widget class="QMainWindow" name="CountDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Count</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/abacus-512.png</normaloff>:/image/abacus-512.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::East</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Number</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLCDNumber" name="lcdNumber">
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>tools</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Transparency</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="pushButtonNumColor">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/image/Color_wheel_dock_icon_by_andybaumgar-d3ezjgc.png</normaloff>:/image/Color_wheel_dock_icon_by_andybaumgar-d3ezjgc.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QSlider" name="horizontalSliderOpacity">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
